app.rb

```diff
@@ -61,7 +64,7 @@
       end
     end
 
-    pull_request.check_commits_style if redmine_issue_repos.include?(pull_request.repo) && pr_action != 'created'
+    pull_request.check_commits_style(redmine_project_for_repo(pull_request.repo)) if pr_action != 'created'
```

Comment:
`pr_action` may be `created` if the event is a comment on a PR (`pull_request_review_comment`), so it avoids re-running the style checks when a comment has been added.